README.md

```diff
@@ -40,11 +40,16 @@ These dependencies must be present before building:
  - `libwnck-3-dev`
 
 
-Use the following command to install the dependencies:
+Use the following command to install the dependencies on Elementary:
```

Comment:
Fixed

Reply:
https://elementary.io/brand It's actually `elementary OS` and not `Elementary` ;)